### PR TITLE
add Juho's topic

### PR DESCRIPTION
**Pull request by salivan**

### CODE REVIEW

File: data/schedule.js [hunk 1]
```diff
@@ -178,15 +178,17 @@ module.exports = {
           "showReadMore": true,
         },
         {
-          "className": "",
+          "className": "juho-vepsalainen",
```
Comment (tyv) [inline data/schedule.js]: think it should be empty, denis was special and has special css

**Review by tyv — CHANGES_REQUESTED** [review]
remove className

File: data/schedule.js [hunk 2]
```diff
@@ -178,15 +178,17 @@ module.exports = {
...
             "hours": "18",
             "minutes": "50"
           },
-          "title": "TBA",
+          "title": "Case Study - React Finland",
           "speakerId": 7,
-          "description": [],
-          "showReadMore": false,
+          "description": [
+          "Organizing a conference means you require a certain amount of technical infrastructure. We developed a custom website and application for <a target=\"_blank\" href='https://react-finland.fi/'>React Finland</a>. In this talk, you'll learn how we did it and how to achieve a modern frontend architecture using technologies such as Webpack, React, and GraphQL. You will also gain insight on the non-technical aspects including marketing and sales and see how they align with technology."
```
Comment (marina-marina) [inline data/schedule.js]: plz add 2 more spaces at the beginning of the line here

**Comment by marina-marina** [comment]
https://github.com/YGLF-Kyiv/website-2018/pull/46

**Pull request closed without merging**